documentation/docs/quickstart/2-install.md

```diff
@@ -1,85 +1,37 @@
-
-### Step 1: Download helper scripts
-
-We have created automated scripts that simplify the process of installing and running Hummingbot with Docker.
-
-* `create.sh`: Creates a new instance of Hummingbot
-* `start.sh`: Starts a stopped Hummingbot instance
-* `update.sh`: Updates Hummingbot
-
-
-Copy the commands below and paste into your Linux terminal to download the scripts and enable permissions.
-
-```
-wget https://raw.githubusercontent.com/CoinAlpha/hummingbot/development/installation/install-docker/install-docker-ubuntu.sh
-wget https://raw.githubusercontent.com/CoinAlpha/hummingbot/development/installation/docker-commands/create.sh
-wget https://raw.githubusercontent.com/CoinAlpha/hummingbot/development/installation/docker-commands/update.sh
-wget https://raw.githubusercontent.com/CoinAlpha/hummingbot/development/installation/docker-commands/start.sh
-chmod a+x *.sh
-```
-
-![helper_scripts](/assets/img/helper_scripts.gif)
-
-Ensure the scripts have been downloaded by running `ls -l` command.
-
-![list_scripts](/assets/img/list_scripts.png)
-
-
-### Step 2: Install Docker
 
-Docker is an open source containerization product that pre-packages all dependencies into a single container, greatly simplifying the installation process. Execute the command below to install Docker using the script we downloaded in the previous step.
+## macOS
 
-```bash
-./install-docker-ubuntu.sh
-```
+  <iframe width="616" height="347" src="https://www.youtube.com/embed/klN-ToclwW4" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen>
+  </iframe>
 
-Close and restart your bash/terminal window to enable the correct permissions for the `docker` command. Otherwise, you may encounter an error in the next steps.
+Step 1. Download Hummingbot from our official website 
 
-We also have instructions for installing Docker on [Debian](/installation/via-docker/linux/#debian) and [CentOS](/installation/via-docker/linux/#centos).
-
-
-### Step 3: Install Hummingbot
-
-Execute `create.sh` script which will help you install an instance of Hummingbot and set up folders to house your logs, configuration files and trades/orders database file.
-
-```
-hummingbot_files       # Default name of top level folder for hummingbot-related files
-├── hummingbot_conf    # Maps to hummingbot's conf/ folder, which stores configuration files
-├── hummingbot_logs    # Maps to hummingbot's logs/ folder, which stores log files
-└── hummingbot_data    # Maps to hummingbot's data/ folder, which stores the SQLite database file
-```
-
-Run command to execute the script.
-
-```
-./create.sh
-```
-
-Follow the prompts to choose a version, give your Hummingbot instance a name, and enter the name of the folder where its files will be saved.
+- [Download the Hummingbot client](https://hummingbot.io/download/)
 
-![script_create](/assets/img/script_create1.gif)
+Step 2. Install Hummingbot on your local computer
```

Comment:
I think we should link to the Youtube video here, so that if we change it later on, we don't need to update this video link.